core/src/main/kotlin/org/evomaster/core/problem/rest/RestPath.kt

```diff
@@ -280,4 +280,16 @@ class RestPath(path: String) {
     private fun encode(s: String): String {
         return URLEncoder.encode(s, "UTF-8")
     }
+
+    fun copy() : RestPath{
+        return RestPath(this.toString())
+    }
+
+    fun getStaticTokens(): List<String> {
```

Comment:
add a comment clarifying what is a "static" token

Reply:
changed another name and added some comments.